### PR TITLE
pcm-1064

### PR DESCRIPTION
**Pull request by sshumake**
@ardasgup @pbathia

### CODE REVIEW

**Comment by pbathia** [comment]
Reviewed.

**Comment by ardasgup** [comment]
Reviewed!

File: app/common/services/constantsService.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ angular.module('RedhatAccess.common').service('ConstantsService', [
...
+            },
+            {
+                ///this refers  in context of "sorting on Case Status (A-Z)"
+                name: gettextCatalog.getString('Case Status (A-Z)'),
+                sortField: 'status',
+                sortOrder: 'ASC'
+            },
+            {
+                ///this refers  in context of "sorting on Case Status (Z-A)"
+                name: gettextCatalog.getString('Case status (Z-A)'),
```
Comment (renujhamtani) [inline app/common/services/constantsService.js]: Case status:  's' should be in CAPS else there would be two seperate stirngs in translations. Never mind i would fix this in some other PR.

**Comment by renujhamtani** [comment]
@sshumake  I am not merging this until we go live on monday as there are some translation stuff in this too.

**Pull request merged**